File: C/1064.c
#include <stdio.h>

int main()
{
    int t;
    float a,b,c,d,e,f,media;
    double total=0;
    scanf("%f%f%f%f%f%f",&a,&b,&c,&d,&e,&f);
    
    if(a>=1){
        t++;
        total+=a;
    };
    if(b>=1){
        t++;
        total+-b;
    };
    if(c>=1){
        t++;
        total+=c;
    };
    if(d>=1){
        t++;
        total+=d;
    };
    if(e>=1){
        t++;
        total+=e;
    };
    if(f>=1){
        t++;
        total+=f;
    };
    total=total/t;
    printf("%i valores positivos\n%.1f\n", t,total);
    return 0;
}

File: Java/1064.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.text.DecimalFormat;

public class Main {

    public static void main(String[] args) throws IOException {
        
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);
        DecimalFormat df = new DecimalFormat("#.#");
        
        int positivos = 0, i;
        float media = 0, n;
        
        for(i = 0; i < 6; i++) {
            n = Float.parseFloat(in.readLine());
            
            if(n > 0) {
                positivos++;
                media += n;
            }
        }
        media /= positivos;
        
        System.out.println(positivos + " valores positivos");
        System.out.println(df.format(media));
    }

}